<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FAT_FATURA_DETAYLARI_BATCH</class>
 <widget class="QDialog" name="FAT_FATURA_DETAYLARI_BATCH">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>286</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" rowstretch="50,5" columnstretch="50,5">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout" rowstretch="1,0,0,0,0,0,0,0" columnstretch="1,0,0,0">
     <item row="2" column="0">
      <widget class="QLabel" name="label_yurtici_satislar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Yurtiçi Satışlar</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QSearchEdit" name="searchEdit_sube_yurtici_satis_hsp_id">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="whatsThis">
        <string>Arama widget. </string>
       </property>
       <property name="QSearchEdit" stdset="0">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="lineEdit_sube_yurtici_satis_hsp_id">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QSearchEdit" name="searchEdit_sube_hizmet_urt_maliyet_hsp_id">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="whatsThis">
        <string>Arama widget. </string>
       </property>
       <property name="QSearchEdit" stdset="0">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="lineEdit_sube_hizmet_urt_maliyet_hsp_id">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_hizmet_uretim_maliyeti">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Hizmet Üretim Maliyeti</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_cari_fis_no">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Cari Fiş No</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_muh_fis_no">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Muhasebe Fiş No</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCommaEdit" name="commaEdit_muh_fis_no">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="whatsThis">
        <string>Sayilari formatina(Or:1,000,234.50) donusturur. </string>
       </property>
       <property name="QCommaEdit" stdset="0">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCommaEdit" name="commaEdit_cari_fis_no">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="whatsThis">
        <string>Sayilari formatina(Or:1,000,234.50) donusturur. </string>
       </property>
       <property name="QCommaEdit" stdset="0">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QCheckBox" name="checkBox_otv_muaf_mi">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Ötv  den muaf mı?</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3" rowspan="2">
      <widget class="QFrame" name="frame_tevkifat">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <widget class="QCommaEdit" name="commaEdit_tevkifat_yuzdesi">
          <property name="whatsThis">
           <string>Sayilari formatina(Or:1,000,234.50) donusturur. </string>
          </property>
          <property name="QCommaEdit" stdset="0">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>100</width>
            <height>100</height>
           </rect>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_tevkifat_yuzdesi">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Tevkifat Yüzdesi      %  </string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="6" column="0" rowspan="2">
      <widget class="QCheckBox" name="checkBox_tevkifatli_fatura_mi">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Tevkifatlı fatura mı?</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QCheckBox" name="checkBox_kdv_muaf_mi">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Kdv den muaf mı?</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="1">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Satıştan İade Hesabı </string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLineEdit" name="lineEdit_satistan_iade_hesap_id">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QSearchEdit" name="searchEdit_satistan_iade_hesap_id">
       <property name="whatsThis">
        <string>Arama widget. </string>
       </property>
       <property name="QSearchEdit" stdset="0">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>677</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widget_batch_buttons" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCommaEdit</class>
   <extends>QLineEdit</extends>
   <header>QCommaEdit.h</header>
  </customwidget>
  <customwidget>
   <class>QSearchEdit</class>
   <extends>QWidget</extends>
   <header>QSearchEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>searchEdit_sube_yurtici_satis_hsp_id</tabstop>
  <tabstop>searchEdit_sube_hizmet_urt_maliyet_hsp_id</tabstop>
  <tabstop>searchEdit_satistan_iade_hesap_id</tabstop>
  <tabstop>checkBox_kdv_muaf_mi</tabstop>
  <tabstop>checkBox_otv_muaf_mi</tabstop>
  <tabstop>checkBox_tevkifatli_fatura_mi</tabstop>
  <tabstop>commaEdit_tevkifat_yuzdesi</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
